oas_docs/overlays/kibana.overlays.yaml

```
@@ -65,10 +65,26 @@ actions:
     description: Change displayName
     update:
       x-displayName: "Machine learning"
+  - target: '$.tags[?(@.name=="roles")]'
+    description: Change displayName and description
+    update:
+      x-displayName: "Roles"
+      description: Manage the roles that grant Kibana privileges.
```

These roles can also grant Elasticsearch privileges. I don't know if "stack" is appropriate, or if "Elasticsearch and Kibana" is more user friendly.

```suggestion
      description: Manage the roles that grant stack privileges.
```

---

Thanks, I used the latter in f3c2f488652b28face5906629ae106057fb29637